docs-source/docs/modules/how-to/examples/cleanup-dependencies-project/build.sbt

```diff
@@ -50,51 +50,51 @@ dockerRepository := sys.props.get("docker.registry")
 ThisBuild / dynverSeparator := "-"
 
 libraryDependencies ++= Seq(
-  "com.typesafe.akka" %% "akka-cluster-sharding-typed" % AkkaVersion,
-  "com.typesafe.akka" %% "akka-stream" % AkkaVersion,
-  // tag::remove-akka-persistence-cassandra[]
-  "com.typesafe.akka" %% "akka-persistence-cassandra" % AkkaPersistenceCassandraVersion,
-  // end::remove-akka-persistence-cassandra[]
+  "org.apache.pekko" %% "pekko-cluster-sharding-typed" % PekkoVersion,
+  "org.apache.pekko" %% "pekko-stream" % PekkoVersion,
+  // tag::remove-pekko-persistence-cassandra[]
+  "org.apache.pekko" %% "pekko-persistence-cassandra" % PekkoPersistenceCassandraVersion,
+  // end::remove-pekko-persistence-cassandra[]
 
-  // tag::add-akka-persistence-jdbc[]
-  "com.lightbend.akka" %% "akka-persistence-jdbc" % AkkaPersistenceJdbcVersion,
-  // end::add-akka-persistence-jdbc[]
+  // tag::add-pekko-persistence-jdbc[]
+  "org.apache.pekko" %% "pekko-persistence-jdbc" % PekkoPersistenceJdbcVersion,
+  // end::add-pekko-persistence-jdbc[]
 
-  // tag::remove-akka-projection[]
-  "com.lightbend.akka" %% "akka-projection-eventsourced" % AkkaProjectionVersion,
-  "com.lightbend.akka" %% "akka-projection-cassandra" % AkkaProjectionVersion,
-  "com.lightbend.akka" %% "akka-projection-jdbc" % AkkaProjectionVersion,
-  "com.typesafe.akka" %% "akka-persistence-query" % AkkaVersion,
-  "com.lightbend.akka" %% "akka-projection-testkit" % AkkaProjectionVersion % Test,
-  // end::remove-akka-projection[]
+  // tag::remove-pekko-projection[]
+  "org.apache.pekko" %% "pekko-projection-eventsourced" % PekkoProjectionVersion,
+  "org.apache.pekko" %% "pekko-projection-cassandra" % PekkoProjectionVersion,
+  "org.apache.pekko" %% "pekko-projection-jdbc" % PekkoProjectionVersion,
+  "org.apache.pekko" %% "pekko-persistence-query" % PekkoVersion,
+  "org.apache.pekko" %% "pekko-projection-testkit" % PekkoProjectionVersion % Test,
+  // end::remove-pekko-projection[]
 
   /*
   // tag::replace-offset-store-for-projections-jdbc[]
-  -  "com.lightbend.akka" %% "akka-projection-cassandra" % AkkaProjectionVersion,
-  +  "com.lightbend.akka" %% "akka-projection-jdbc" % AkkaProjectionVersion,
+  -  "org.apache.pekko" %% "pekko-projection-cassandra" % PekkoProjectionVersion,
+  +  "org.apache.pekko" %% "pekko-projection-jdbc" % PekkoProjectionVersion,
   // end::replace-offset-store-for-projections-jdbc[]
    */
-  "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion,
+  "org.apache.pekko" %% "pekko-http" % PekkoHttpVersion,
   // tag::remove-grpc-optional[]
-  "com.typesafe.akka" %% "akka-http2-support" % AkkaHttpVersion,
+  // "org.apache.pekko" %% "pekko-http2-support" % PekkoHttpVersion,
```

Comment:
well I didn't find pekko-http2-support，Is this included in pekko http or not published ?
@pjfanning

Reply:
not published - not needed for pekko (code was moved to pekko-http-core)